index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Henary's Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div>
+            <a href = "index.html" class="brand">My Logo</a>
+        </div>
+        <nav>
+            <img class="menuIcon" src="images/menuIcon.png" alt=" menu">
+        </nav>
+    </header>
+    <main>
+        <section class="headline">
+            <h1 class="introduce">I’m Henary
+                Glad to see you!
+            </h1>
+
+             <p class="discription">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+             
+             <div class="social">
+                <p class="contact">LET’S CONNECT</p>
+                <ul class="social-connection">
+                    <li><a href="https://twitter.com/HenaryKhuraijam"><img class="social-icon"src="images/twiter.png" alt="twiter"></a></li>
+                    <li><a href="https://www.linkedin.com/in/henary-khuraijam-50487317a/"><img class="social-icon"src="images/Linkedin icon.png" alt="Linkedin"></a></li>
+                    <li><a href="#"><img class="social-icon"src="images/M.png" alt="M"></a></li>
+                    <li><a href="https://github.com/HenaryKhuraijam"><img class="social-icon"src="images/github.png" alt="github"></a></li>
+                    <li><a href="#"><img class="social-icon"src="images/V.png" alt="V"></a></li>
```

Comment:
- [ ] [OPTIONAL] You have done a great job putting the attribute names :+1: . But the names are not descriptive enough to be [Semantically](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjB5KvKkcf8AhUyY6QEHQNeAXAQFnoECAoQAQ&url=https%3A%2F%2Fwww.w3schools.com%2Fhtml%2Fhtml5_semantic_elements.asp&usg=AOvVaw38SC2etR0JeORCKyF1fqbU) correct. For example: instead of using `M`, it would be of better practice to use `medium` to ensure easy screen readers accessibility.